r/R/dataset-write.R

```diff
@@ -53,6 +53,17 @@
 #' - `codec`: A [Codec] which will be used to compress body buffers of written
 #'   files. Default (NULL) will not compress body buffers.
 #' @return The input `dataset`, invisibly
+#' @examples
+#' \donttest{
+#' # we can group by cyl, cyl and gear or even more variables
+#' write_dataset(mtcars, tempdir(), "feather", partitioning = "cyl"))
+#' write_dataset(mtcars, tempdir(), "feather", partitioning = c("cyl", "gear"))
+#'
+#' # the latter example is the same as the following dplyr chained statement
+#' # mtcars %>%
+#' #  group_by(cyl, gear) %>%
+#' #  write_dataset(dout2, "feather")
+#' }
```

Comment:
I think it would be good to have one more example just like these but with `hive_style = FALSE` and print its directory contents so that the resulting difference between the two ways is clear.